### PR TITLE
refactor(scan): ensure we use the parsed election

### PR DESCRIPTION
**Pull request by eventualbuddha**
Apparently TS favors the `.json` file when importing, so the `election.ts` files were being ignored. Go figure!

### CODE REVIEW

File: apps/module-scan/src/interpreter.test.ts [hunk 1]
```diff
@@ -60,6 +60,8 @@ test('extracts votes encoded in a QR code', async () => {
           markThresholds: { definite: 0.2, marginal: 0.17 },
         },
         testMode: true,
+        adjudicationReasons:
```
Comment (carolinemodic) [inline apps/module-scan/src/interpreter.test.ts:63]: Is this supposed to be dependent on another PR? CircleCi is complaining about centralScanAdjudicationReasons not existing in the election type?

Comment (eventualbuddha) [reply apps/module-scan/src/interpreter.test.ts:63]: I extracted this commit from my work on #726, so yeah that wasn't supposed to be included.

**Comment by eventualbuddha** [comment]
Ready to go again, @carolinemodic.

**Comment by eventualbuddha** [comment]
Ping @carolinemodic

**Review by carolinemodic — APPROVED** [review]
sorry! thought I had approved this for some reason
